src/postgraphql/withPostGraphQLContext.ts

```diff
@@ -0,0 +1,223 @@
+import createDebugger = require('debug')
```

Comment:
Refactored `withPostGraphQLContext` to add types, and I moved `debugPgClient` and `setupPgClientTransaction` to this file because they weren’t being used anywhere else 🤷‍♂️